src/api/mod.rs

```diff
@@ -1390,6 +1413,71 @@ impl<'a> AuthenticatedApi<'a> {
     }
 }
 
+/// Available datasets for fetching organization events
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Dataset {
+    /// Our logs dataset
+    OurLogs,
+}
+
+impl Dataset {
+    /// Returns the string representation of the dataset
+    pub fn as_str(&self) -> &'static str {
```

Comment:
I believe you can make this private

```suggestion
    fn as_str(&self) -> &'static str {
```